plugin/code_lens.py

```diff
@@ -150,7 +150,8 @@ def render(self, mode: str) -> None:
             for key, group in self._code_lenses.items():
                 region = sublime.Region(*key)
                 phantom_region = self._get_phantom_region(region)
-                html = '\n<small style="font-family: system">|</small>\n'.join(lens.small_html for lens in group)
+                html = '<body id="lsp-code-lens">{}</body>'.format(
```

Comment:
Why `id` instead of the `class` attribute?

Reply:
Because it is the recommended way to do it: https://www.sublimetext.com/docs/minihtml.html#best_practices

And maybe it helps to avoid accidentally using it here in this package to style the phantoms (if we want to do that, we could wrap the inner html into a div tag with another class and use that class to add styles).